trino-aws-proxy/src/main/java/io/trino/aws/proxy/server/signing/SigningQueryParameters.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.aws.proxy.server.signing;
+
+import com.google.common.collect.ImmutableSet;
+import io.trino.aws.proxy.spi.collections.ImmutableMultiMap;
+import io.trino.aws.proxy.spi.collections.MultiMap;
+
+import java.util.Set;
+
+public final class SigningQueryParameters
+{
+    public static final Set<String> SIGNING_QUERY_PARAMETERS = ImmutableSet.of(
+            "X-Amz-Date",
+            "X-Amz-Algorithm",
+            "X-Amz-Credential",
+            "X-Amz-SignedHeaders",
+            "X-Amz-Signature",
+            "X-Amz-Expires");
+
+    public record SplitQueryParameters(MultiMap forwardableQueryParameters, MultiMap signingQueryParameters)
```

Comment:
For me, `forwardableQueryParameters` is confusing. Can we have a different name? Something more like pass-through or ignorable.

Reply:
`pass-through` sounds good, I don't think ignorable would be a good name as the point of these is precisely that we ignore them in the proxy but they are the "important" ones that get sent unmodified to S3